Issue
  { issueClosedAt = Just 2013 (-08) (-07) 22 : 23 : 27 UTC
  , issueUpdatedAt = 2013 (-08) (-07) 22 : 26 : 04 UTC
  , issueEventsUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/1205/events"
  , issueHtmlUrl =
      Just "https://github.com/rethinkdb/rethinkdb/issues/1205"
  , issueClosedBy = Nothing
  , issueLabels = []
  , issueNumber = 1205
  , issueAssignee = Nothing
  , issueUser =
      SimpleUser
        { simpleUserId = Id 1526797
        , simpleUserLogin = N "tuhinc"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/1526797?v=3"
        , simpleUserUrl = "https://api.github.com/users/tuhinc"
        , simpleUserType = OwnerUser
        }
  , issueTitle = "Handshake timedout issue"
  , issuePullRequest = Nothing
  , issueUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/1205"
  , issueCreatedAt = 2013 (-07) (-26) 23 : 04 : 16 UTC
  , issueBody =
      Just
        "Hey guys -- love rethink, you guys are awesome. I'm running into an issue that could be related to the client drivers.\n\nI got the following error when running npm install rethinkdb:\n\n  CXX(target) Release/obj.target/protobuf/protobuf.o\n../protobuf.cpp:7:10: fatal error: 'google/protobuf/dynamic_message.h' file not found\n# include <google/protobuf/dynamic_message.h>\n\n```\n     ^\n```\n\n1 error generated.\nmake: **\\* [Release/obj.target/protobuf/protobuf.o] Error 1\ngyp ERR! build error \ngyp ERR! stack Error: `make` failed with exit code: 2\ngyp ERR! stack     at ChildProcess.onExit (/opt/boxen/nodenv/versions/v0.8.8/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:219:23)\ngyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:91:17)\ngyp ERR! stack     at Process._handle.onexit (child_process.js:674:10)\ngyp ERR! System Darwin 12.3.0\ngyp ERR! command \"node\" \"/opt/boxen/nodenv/versions/v0.8.8/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js\" \"rebuild\"\ngyp ERR! cwd /Users/hackreactor/node_modules/rethinkdb/node_modules/node-protobuf\ngyp ERR! node -v v0.8.8\ngyp ERR! node-gyp -v v0.6.7\ngyp ERR! not ok \nnpm WARN optional dep failed, continuing node-protobuf@1.0.5\n---\n\nbut from what I read from previous issues it seems this error does not affect driver installation. however, i am unable to connect to my database from inside node's repl.\n\nafter r = require('rethinkdb')\nand r.connect({host: 'localhost', port:28015}, function(err, conn) { if (err) throw err; else{ console.log(conn);}});\n\nI get the following error: \n\ntimers.js:103\n            if (!process.listeners('uncaughtException').length) throw e;\n                                                                      ^\nRqlDriverError: Handshake timedout\n---\n\nI know the server is running as I am able to access it from my browser by pointing it to 127.0.0.1:8080\n\nThanks so much guys! Sincerely appreciate you guys working on this --\n"
  , issueState = "closed"
  , issueId = Id 17289007
  , issueComments = 29
  , issueMilestone =
      Just
        Milestone
          { milestoneCreator =
              SimpleUser
                { simpleUserId = Id 706854
                , simpleUserLogin = N "AtnNn"
                , simpleUserAvatarUrl =
                    "https://avatars.githubusercontent.com/u/706854?v=3"
                , simpleUserUrl = "https://api.github.com/users/AtnNn"
                , simpleUserType = OwnerUser
                }
          , milestoneDueOn = Nothing
          , milestoneOpenIssues = 0
          , milestoneNumber = 19
          , milestoneClosedIssues = 175
          , milestoneDescription =
              Just
                "It's a feature. The issue describes a RethinkDB feature or design choice as if it was a bug."
          , milestoneTitle = "notabug"
          , milestoneUrl =
              "https://api.github.com/repos/rethinkdb/rethinkdb/milestones/19"
          , milestoneCreatedAt = 2013 (-03) (-29) 21 : 07 : 05 UTC
          , milestoneState = "closed"
          }
  }